data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+</div>
+
+<div type="textpart" subtype="section" n="24">
+<p>And if any one were to desire to use more undisguised terms, he would not call the world, which is perceptible only to the intellect, any thing else but the reason of God, already occupied in the creation of the world; for neither is a city, while only perceptible to the intellect, anything else but the reason of the architect, who is already designing to build one perceptible to the external senses, on the model of that which is so only to the intellect—</p>
+</div>
+
+<div type="textpart" subtype="section" n="25">
+<p>this is the doctrine of Moses, not mine. Accordingly he, when recording the creation of man, in words which follow, asserts expressly, that he was made in the image of God—and if the image be a part of the image, then manifestly so is the entire form, namely, the whole of this world perceptible by the external senses, which is a greater imitation of the divine image than the human form is. It is manifest also, that the archetypal seal, which we call that world which is perceptible only to the intellect, must itself be the archetypal model, the idea of ideas, the Reason of God.
+<milestone unit="chapter" n="7"/></p>
+</div>
+
+<div type="textpart" subtype="section" n="26">
+<p>Moses says also; "In the beginning God created the heaven and the earth:" taking the beginning to be, not as some men think, that which is according to time; for before the world time had no existence, but was created either simultaneously with it, or after it; for since time is the interval of the motion of the heavens, there could not have been any such thing as motion before there was anything which could be moved; but it follows of necessity that it received existence subsequently or simultaneously. It therefore follows also of necessity, that time was created either at the same
+<pb n="7"/>
+moment with the world, or later <corr>than it—and</corr> to venture to assert that it is older than the world is absolutely inconsistent with philosophy.</p>
+</div>
+
+<div type="textpart" subtype="section" n="27">
+<p>But if the beginning spoken of by Moses is not to be looked upon as spoken of according to time, then it may be natural to suppose that it is the beginning according to number that is indicated; so that, "In the beginning he created," is equivalent <corr>to "first</corr> of all he created the heaven;" for it is natural in reality that that should have been the first object created, being both the best of all created things, and being also made of the purest substance, because it was destined to be the most holy abode of the visible Gods who are perceptible by the external senses;</p>
```

Comment:
`<q>` instead of quotation marks, but if quotation marks preserved, should be curly quotes